workspace/assets/tasks/jshint.js

```diff
@@ -47,7 +48,8 @@ module.exports = function jshint (grunt) {
 					App: true,
 					Loader: true,
 					_: true,
-					module: true
+					module: true,
+					google: true
```

Comment:
@PascalPiche wanted to add our `CSS3` module. Do you agree @fhamon ?

Reply:
Sure !